sbin/uta-load

```diff
@@ -34,22 +32,13 @@ etc/scripts/create-new-schema.sh "$source_uta_v" "$loading_uta_v"
 ## for now set up Alembic for schema migrations
 alembic -c etc/alembic.ini upgrade head
 
-# generate seqinfo files from exonsets (this step requires seqrepo)
```

Comment:
The note here reminded me why I put this step here. This step requires seqrepo, which is why it was in uta-load instead of uta-extract. Currently uta-load runs after seqrepo-load, and seqrepo-load needs to run after uta-extract because uta-extract includes the moving of the fasta files.